modules/op-using-remote-pipeline-annotations-with-pipelines-as-code.adoc

```diff
@@ -21,3 +21,25 @@ You can share a pipeline definition across multiple repositories by using the re
 You can reference only one pipeline definition using the annotation.
 ====
 
+[id="overriding-tasks-in-remote-pipeline_{context}"]
+== Overriding a task in a remote pipeline
+
+By default, if you use a remote pipeline annotation in a pipeline run, {pac} uses all the tasks that are a part of the remote pipeline.
+
+To override a task, you can add an annotation for a task with the same name to the remote pipeline annotation.
```

Comment:
nit: task with the same name as in remote pipeline to the pipelinerun annotation